<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRequests</class>
 <widget class="QDialog" name="UserRequests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>351</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QCommandLinkButton" name="homepb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>C:\Users\HP\Desktop\BankClientTest</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Return To Home Page</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/HP/Desktop/home_553416.png</normaloff>C:/Users/HP/Desktop/home_553416.png</iconset>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>481</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>OurServices</string>
     </property>
     <widget class="QPushButton" name="accNumpb">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Account Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newTransactionpb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Transaction</string>
      </property>
      <property name="icon">
       <iconset theme="address-book-new"/>
      </property>
     </widget>
     <widget class="QPushButton" name="accBalancepb">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Account Balance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moneyTransferpb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Money Transfer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="historypb">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transaction History</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_5</zorder>
    <zorder>homepb</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>381</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>GetAccountNumber</string>
     </property>
     <widget class="QLineEdit" name="accNumle">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Your account number is:</string>
      </property>
     </widget>
    </widget>
    <widget class="QCommandLinkButton" name="numtohomepb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/HP/Desktop/previous_189254.png</normaloff>C:/Users/HP/Desktop/previous_189254.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>GetAccountBalance</string>
     </property>
     <widget class="QLabel" name="accNumlbl">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Your account balance is:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="accBalancele">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </widget>
    <widget class="QCommandLinkButton" name="balancetohomepb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/HP/Desktop/previous_189254.png</normaloff>C:/Users/HP/Desktop/previous_189254.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QCommandLinkButton" name="historytohomepb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/HP/Desktop/previous_189254.png</normaloff>C:/Users/HP/Desktop/previous_189254.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="countlbl">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Count:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="countle">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="historyenterpb">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="historyTable">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QCommandLinkButton" name="newtransactiontohomepb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/HP/Desktop/previous_189254.png</normaloff>C:/Users/HP/Desktop/previous_189254.png</iconset>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>371</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>NewTransaction</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transaction:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Deposit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Withdraw</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="transactionamountle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="usertransactionenterpb">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QCommandLinkButton" name="moneytransfertohomepb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/HP/Desktop/previous_189254.png</normaloff>C:/Users/HP/Desktop/previous_189254.png</iconset>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>371</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>MoneyTransfer</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Account Number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="transferamountle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="transferenterpb">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="accnumfortransferle">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
